html/components/_masthead.scss

```diff
@@ -28,11 +31,13 @@
   }
 }
 
-// Added via JS when page is scolled past masthead
+///	Sets styles for the Masthead when the user scrolls past it.
```

Comment:
Do we want to be consistent in verbiage as far as "Sets" "Adds" "Add" when referring to the styles?

Reply:
Yeah, i vote we just go with "Adds".